interview-questions.md

```diff
@@ -1,57 +1,83 @@
- #ASSESSMENT 2: Interview Practice Questions
+# ASSESSMENT 2: Interview Practice Questions
 
 Answer the following questions.
 
 First, without external resources. Challenge yourself to answer from memory as if you were in a job interview.
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-1. What is the difference between a parameter and an argument?
+1. What is the difference between a parameter and an argument? 
 
-Your answer:
+Your answer: Parameter is a variable, argument is the data that you are trying to convey and pass. 
```

Comment:
Make sure to expand on your answers whenever possible.  This should reflect how you would answer the question if asked in an interview.